<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>ConfigWindow</class>
    <widget class="QDialog" name="configWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>480</width>
                <height>270</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Dialog</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
            <item row="3" column="0">
                <widget class="QDialogButtonBox" name="buttonBox">
                    <property name="standardButtons">
                        <set>
                            QDialogButtonBox::Apply|QDialogButtonBox::Close|QDialogButtonBox::Ok|QDialogButtonBox::Reset
                        </set>
                    </property>
                </widget>
            </item>
            <item row="2" column="0">
                <widget class="QTabWidget" name="tabWidget">
                    <property name="currentIndex">
                        <number>0</number>
                    </property>
                    <widget class="QWidget" name="audioTab">
                        <attribute name="title">
                            <string>Audio</string>
                        </attribute>
                        <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,1,0,1">
                            <item row="0" column="0">
                                <widget class="QLabel" name="audioDeviceLabel">
                                    <property name="text">
                                        <string>Device</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="2" column="0">
                                <widget class="QLabel" name="audioSampleRateLabel">
                                    <property name="text">
                                        <string>Sample rate</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="1" column="0">
                                <widget class="QLabel" name="audioChannelsLabel">
                                    <property name="text">
                                        <string>Channels</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="2" column="1">
                                <widget class="QComboBox" name="audioSampleRateComboBox"/>
                            </item>
                            <item row="1" column="1">
                                <widget class="QComboBox" name="audioChannelsComboBox"/>
                            </item>
                            <item row="4" column="0">
                                <spacer name="verticalSpacer">
                                    <property name="orientation">
                                        <enum>Qt::Vertical</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>20</width>
                                            <height>40</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item row="2" column="2">
                                <widget class="QLabel" name="audioSampleSizeLabel">
                                    <property name="text">
                                        <string>Sample size</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="2" column="3">
                                <widget class="QComboBox" name="audioSampleSizeComboBox"/>
                            </item>
                            <item row="0" column="1" colspan="3">
                                <widget class="QComboBox" name="audioDeviceComboBox"/>
                            </item>
                            <item row="1" column="2">
                                <widget class="QLabel" name="audioSampleTypeLabel">
                                    <property name="text">
                                        <string>Sample type</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="1" column="3">
                                <widget class="QComboBox" name="audioSampleTypeComboBox"/>
                            </item>
                        </layout>
                    </widget>
                    <widget class="QWidget" name="outputTab">
                        <attribute name="title">
                            <string>Output</string>
                        </attribute>
                        <layout class="QGridLayout" name="outputTabGridLayout" columnstretch="0,1,0,1">
                            <item row="0" column="0">
                                <widget class="QLabel" name="audioOutputDeviceLabel">
                                    <property name="text">
                                        <string>Device</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="2" column="0">
                                <widget class="QLabel" name="audioOutputSampleRateLabel">
                                    <property name="text">
                                        <string>Sample rate</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="1" column="0">
                                <widget class="QLabel" name="audioOutputChannelsLabel">
                                    <property name="text">
                                        <string>Channels</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="2" column="1">
                                <widget class="QComboBox" name="audioOutputSampleRateComboBox"/>
                            </item>
                            <item row="1" column="1">
                                <widget class="QComboBox" name="audioOutputChannelsComboBox"/>
                            </item>
                            <item row="4" column="0">
                                <spacer name="verticalSpacer">
                                    <property name="orientation">
                                        <enum>Qt::Vertical</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>20</width>
                                            <height>40</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                            <item row="2" column="2">
                                <widget class="QLabel" name="audioOutputSampleSizeLabel">
                                    <property name="text">
                                        <string>Sample size</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="2" column="3">
                                <widget class="QComboBox" name="audioOutputSampleSizeComboBox"/>
                            </item>
                            <item row="0" column="1" colspan="3">
                                <widget class="QComboBox" name="audioOutputDeviceComboBox"/>
                            </item>
                            <item row="1" column="2">
                                <widget class="QLabel" name="audioOutputSampleTypeLabel">
                                    <property name="text">
                                        <string>Sample type</string>
                                    </property>
                                </widget>
                            </item>
                            <item row="1" column="3">
                                <widget class="QComboBox" name="audioOutputSampleTypeComboBox"/>
                            </item>
                        </layout>
                    </widget>
                </widget>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>
